There will be a list with links to modules

<!DOCTYPE html>
<html>
	<head>
		<title>Modules</title>
	</head>
	<body>
		<h1>Modules in Y3S2</h1>

		<a href='#BS32004'><p>Molecular Microbiology</p></a>
                <a href='#BS32011'><p>Practical Project A - Bioinformatics</p></a>
                <a href='#module3'><p>Applied Bioinformatics (Block D)</p></a>
                <a href='#module4'><p>Practical Project B - Synthetic Biology (Block D)</p></a>

	<table border='1' style='width:100%'>
	<tr>
		<th>Module Name</th>
		<th>Description</th>
	</tr>
	<tr>
		<td>BS32004:<a name='BS32004'></a></td>
		<td><p>The aim of this Module is to introduce students to specialist topics within the areas of bacterial and fungal microbiology. This module will underpin more specialised areas of molecular microbiology that the students will encounter in their Level 4 laboratory projects and in Semester 2 of Level 4. The module aims to strengthen students' skills in problem solving, critical analysis of scientific literature, and in self-directed learning.
		After study of lecture and associated materials, students will be able to explain broad aspects of microbiology and relate this to protein structure / function and molecular genetics. Students will understand the principles that underlie the processes of how bacteria sense, respond and move to environmental stimuli, nutrient sensing and uptake by bacteria, how, why and when antibiotics are produced by bacteria, the bacterial cytoskeleton, molecular motors, and the process of cell division in prokaryotes, multicellular and community behaviour in bacteria and fungi, geomicrobiology and its applications, fungal cell wall biosynthesis and the molecular basis of pathogenicity and virulence in bacteria.</p></td>
	</tr>

	<tr>
		<td>BS32011:<a name='BS32011'></a></td>
		<td><p>3rd Year Bioinformatics practical
		Aims
		1 To develop the practical skills required to effectively implement bioinformatics workflows and data analysis. 1 To develop understanding of how to work collaboratively on data analysis 1 to understand provenance and sustainability in bioinformatics analysis
		Structure
		The practical will consist of two parts. The first part is an intensive refresher on Python. We will also introduce using version control with git so you will need to create for yourselves a github account before the first session.
		To get a refresher in Python before the practical starts, you can work through some of the on-line courses, eg http://pythonforbiologists.com (you will need to register to get the data files for the examples - it is free) or http://trypython.org.
		Over the course of the project we will * 
		Retrieve some gene expression data from GEO, * write a script that will 
		process it into a suitable form. * Place it into a relational database. 
		* Write a web script that can retrieve and display the data and set it 
		up as a web server. We will record our progress using version control 
		and document our code well. working platform We will use the Scientific 
		Computing Lab. This is a virtual environment where we can log in to a 
		remote desktop. A copy of this environment will be available as a 
		virtual machine so you can install it on your own machine(s). You'll 
		discover how easy it is to keep these in sync using git/Github. Project 
		timeline Lab meetings Initial introductory sessions will be Monday 19th 
		January 12-3pm, Wednesday 21st 10-1, and Thursday 22nd 11-2 and 3-6pm. 
		You are expected to work outside these hours - the contact time is to 
		help you get up to speed with the nature of the practical. Following the 
		introductory week, lab meetings will be from 12.00-13.00 on Wednesdays 
		in Tower IT suite B from week 16 (28th Jan)-Week 19 Timelines are as 
		follows with week by week deliverables noted. Deliverables should be 
		tagged in your GitHub repository.</p></td>
	</tr>

	<tr>
		<td>Module 3:<a name='module3'></a></td>
		<td><p>Will be applied bioinformatics</p></td>
	</tr>

	<tr>
		<td>Module 4:<a name='module4'></a></td>
		<td><p>Will be practical project in synthetic biology</p></td>
	</tr>
	</table>

	</body>
</html>
